packages/ember-runtime/lib/computed/reduce_computed_macros.js

```diff
@@ -13,7 +13,7 @@ import { isArray } from 'ember-runtime/utils';
 
 function reduceMacro(dependentKey, callback, initialValue) {
   return computed(`${dependentKey}.[]`, function() {
-    return get(this, dependentKey).reduce(callback, initialValue);
+    return get(this, dependentKey).reduce(callback.bind(this), initialValue);
```

Comment:
are we not using 2.0? I guess our users may also not be, i'll quickly implement it the lame way.